tests/test_dialogues.py

```diff
@@ -31,8 +38,8 @@ def test_inmemory_tracker_store(filename):
 
 
 def test_tracker_restaurant():
-    domain = Domain.load("data/test_domains/default_with_slots.yml")
-    filename = 'data/test_dialogues/enter_name.json'
+    domain = Domain.load("examples/restaurantbot/restaurant_domain.yml")
+    filename = 'data/test_dialogues/restaurantbot.json'
     tracker = tracker_from_dialogue_file(filename, domain)
-    assert tracker.get_slot("name") == "holger"
-    assert tracker.get_slot("location") is None     # slot doesn't exist!
+    assert tracker.get_slot("price") == "lo"
```

Comment:
should that be `low`?

Reply:
This is how it is mapped in the nlu data - `[cheap](price:lo)`

Reply:
There is also `[expensive](price:hi`